articles/data-factory/source-control.md

```diff
@@ -20,7 +20,7 @@ By default, the Azure Data Factory user interface experience (UX) authors direct
 
 - The Data Factory service doesn't include a repository for storing the JSON entities for your changes. The only way to save changes is via the **Publish All** button and all changes are published directly to the data factory service.
 - The Data Factory service isn't optimized for collaboration and version control.
-- The Azure Resource Manager template required to deploy Data Factory itself is not included in this integration. Rather the integration assumes the Data Factory itself has been deployed and this integration applies to everything defined in the Data Factory.
+- The Azure Resource Manager template required to deploy Data Factory itself is not included. Rather the integration assumes the Data Factory itself has been deployed and this integration applies to everything defined in the Data Factory.
```

Comment:
Can you remove this text as well "Rather the integration assumes the Data Factory itself has been deployed and this integration applies to everything defined in the Data Factory.". Thank you.